bitmagnet.io/setup/installation.md

```diff
@@ -61,6 +59,26 @@ services:
 
 After running `docker compose up -d` you should be able to access the web interface at `http://localhost:3333`. The DHT crawler should have started and you should see items appear in the web UI within around a minute.
 
+## TrueNas Scale
+
+To effortlessly kickstart your journey with bitmagnet, integrate the [TrueCharts catalog](https://truecharts.org/manual/SCALE/guides/getting-started#adding-truecharts) and proceed with app installation on TrueNAS Scale.
+
+Optionally, you can configure the `TMDB API Key` by following the guidelines outlined [here](configuration.md#obtaining-a-tmdb-api-key).
+
+![Bitmagnet Config Options](/assets/images/bitmagnent-scale-app-config.png)
+
+Worth noting, the PostgreSQL database is seamlessly configured and installed alongside the app. Just ensure to meticulously adhere to the getting started guide, installing all the recommended operators before initiating the app installation process.
+
+![Bitmagnet Scale App](/assets/images/bitmagnet-scale-app.png)
+
+### Addons
+
+We offer a vpn addon by Gluetun; which is super easy to configure per app; just follow our guide [here](https://truecharts.org/manual/SCALE/guides/vpn-setup).
+
+Make sure to follow Gluetun's upstream docs for your vpn serivice; we recommend to setup gluetun by env variabes.
+
+![TrueCharts Gluetun Addon](/assets/images/truechart-gluetun-addon.png)
```

Comment:
@xstar97 This should basically not be here.

We would also not explicitly include this in guides we make ourselves and it makes it impossible to maintain.